code/src/App.js

```diff
@@ -1,9 +1,43 @@
-import React from 'react';
+/* eslint-disable */
+import React, { useState } from 'react';
+import { Books } from 'components/Books';
+import { Result } from 'components/Result';
+import { Language } from 'components/Language';
+import { Version } from 'components/Version';
+import { Header } from 'components/Header';
 
 export const App = () => {
+  const [step, setStep] = useState(1);
+  const [book, setBooks] = useState(false);
+  const [language, setLanguage] = useState('');
+  const [version, setVersion] = useState('');
+
+  const handleStepIncrease = () => {
+    setStep(step + 1);
+  }
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <Header />
+      {step === 1 && (
+        <Books book={book} setBooks={setBooks} inputLabel="Which books would you like to set up on your wishlist?" />
+      )}
+      {step === 2 && (
+        <Language language={language} setLanguage={setLanguage} inputLabel="What language do you want?" />
+      )}
+      {step === 3 && (
+        <Version version={version} setVersion={setVersion} inputLabel="Which version would you like" />
+      )}
+      {step >= 4 && (
+        <Result book={book} language={language} version={version} />
+      )}
+      {step < 4 && (
```

Comment:
It would be nice if you had the time to make another component (another page) that you get to after pressing submit. Just a simple confirmation that the email has been sent 👍